src/OpenLoco/Date.cpp

```diff
@@ -159,396 +159,396 @@ namespace OpenLoco
-            { MonthId::october, 20 },
-            { MonthId::october, 21 },
-            { MonthId::october, 22 },
-            { MonthId::october, 23 },
-            { MonthId::october, 24 },
-            { MonthId::october, 25 },
-            { MonthId::october, 26 },
-            { MonthId::october, 27 },
-            { MonthId::october, 28 },
-            { MonthId::october, 29 },
-            { MonthId::october, 30 },
-            { MonthId::october, 31 },
-            { MonthId::november, 1 },
-            { MonthId::november, 2 },
-            { MonthId::november, 3 },
-            { MonthId::november, 4 },
-            { MonthId::november, 5 },
-            { MonthId::november, 6 },
-            { MonthId::november, 7 },
-            { MonthId::november, 8 },
-            { MonthId::november, 9 },
-            { MonthId::november, 10 },
-            { MonthId::november, 11 },
-            { MonthId::november, 12 },
-            { MonthId::november, 13 },
-            { MonthId::november, 14 },
-            { MonthId::november, 15 },
-            { MonthId::november, 16 },
-            { MonthId::november, 17 },
-            { MonthId::november, 18 },
-            { MonthId::november, 19 },
-            { MonthId::november, 20 },
-            { MonthId::november, 21 },
-            { MonthId::november, 22 },
-            { MonthId::november, 23 },
-            { MonthId::november, 24 },
-            { MonthId::november, 25 },
-            { MonthId::november, 26 },
-            { MonthId::november, 27 },
-            { MonthId::november, 28 },
-            { MonthId::november, 29 },
-            { MonthId::november, 30 },
-            { MonthId::december, 1 },
-            { MonthId::december, 2 },
-            { MonthId::december, 3 },
-            { MonthId::december, 4 },
-            { MonthId::december, 5 },
-            { MonthId::december, 6 },
-            { MonthId::december, 7 },
-            { MonthId::december, 8 },
-            { MonthId::december, 9 },
-            { MonthId::december, 10 },
-            { MonthId::december, 11 },
-            { MonthId::december, 12 },
-            { MonthId::december, 13 },
-            { MonthId::december, 14 },
-            { MonthId::december, 15 },
-            { MonthId::december, 16 },
-            { MonthId::december, 17 },
-            { MonthId::december, 18 },
-            { MonthId::december, 19 },
-            { MonthId::december, 20 },
-            { MonthId::december, 21 },
-            { MonthId::december, 22 },
-            { MonthId::december, 23 },
-            { MonthId::december, 24 },
-            { MonthId::december, 25 },
-            { MonthId::december, 26 },
-            { MonthId::december, 27 },
-            { MonthId::december, 28 },
-            { MonthId::december, 29 },
-            { MonthId::december, 30 },
-            { MonthId::december, 31 },
+        static constexpr std::pair<Month, uint8_t> month_table[] = {
+            { Month::January, 1 },
```

Comment:
I think you can write a constexpr function to return these and save the hardcoding